src/middleware/authentication/middleware.ts

```diff
@@ -0,0 +1,7 @@
+import { NextFunction, Request, Response } from "express";
+
+async function main(request: Request, response: Response, next: NextFunction) {
```

Comment:
This is where we should verify the JWT key, correct?

Reply:
Correct!